client/my-sites/checkout/checkout-thank-you/standard-checkout-thank-you.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import { Card, ConfettiAnimation } from 'calypso/../packages/components/src';
```

Comment:
I'm sure this was auto-generated, but we should not import directly from a package path.

```suggestion
import { Card, ConfettiAnimation } from '@automattic/components';
```

Reply:
Thanks for catching this 🙏